### PR TITLE
feat: added avatar and badge

### PR DESCRIPTION
**Pull request by AtulPant2704**
This is just the layout of the component.

### CODE REVIEW

**Comment by AnkurChunekar** [comment]
Great work Atul, 
All of your components look great, code is well indented and is very clean.

**Review by Abhijeet43 — APPROVED** [review]
Hey Atul the code looks good and naming convention is also well specified

File: components/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+.cart-badge .badge-count {
+    position: absolute;
+}
+
+.icon-badge {
+    position: relative;
+}
+
+.icon-badge .badge-count {
```
Comment (mohammed-sas) [inline components/Badge/badge.css]: you could specify "position:absolute" in ".badge-count" itself

File: components/Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    border-radius: 50%;
+    top: -25%;
+    right: -25%;
+}
+
+.cart-badge {
+    position: relative;
+}
+
+.cart-badge .badge-count {
```
Comment (mohammed-sas) [inline components/Badge/badge.css]: specify postion:absolute in badge-count class

**Review by AnkurChunekar — APPROVED** [review]

File: components/Badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+/* Styling badge color and position */
+
+.badge-off {
+    position: relative;
+
+}
+
+.badge-off .round-badge {
+    position: absolute;
```
Comment (mohammed-sas) [inline components/Badge/badge.css]: specify the position in .round-badge class and specify only background color over here . similarly for badge-on and badge-away too

**Review by mohammed-sas — CHANGES_REQUESTED** [review]
making those would reduce repetitions

**Review by mohammed-sas — APPROVED** [review]

**Review by mohammed-sas — APPROVED** [review]

**Review by mohammed-sas — APPROVED** [review]

**Review by mohammed-sas — APPROVED** [review]

**Pull request merged**